core/css.js

```diff
@@ -472,13 +472,13 @@ Blockly.Css.CONTENT = [
     'opacity: 0;',
   '}',
 
-  '.blocklyScrollbarKnob {',
-    'fill: $colour_scrollbar;',
+  '.blocklyScrollbarHandle {',
+    'fill: #ccc;',
```

Comment:
Note - this seems like a merge problem overwriting something we put into Scratch-blocks ($coulour_scrollbar*).